src/wrapper/simulation_objects.py

```diff
@@ -84,3 +86,222 @@ def update(self, data: dict):
 
     def add_subscriptions(self) -> int:
         return 0
+
+
+class Switch(Node):
+    """A switch in the simulation which can point either left or right
+    (see `ISimulationSwitch.State`)
+    """
+
+    class State(Enum):
+        """The possible states of the switch"""
+
+        LEFT = 1
+        RIGHT = 2
+
+    state: State
+
+
+class Track(SimulationObject):
+    """A track in the simulation where trains can drive along"""
+
+    blocked: bool
+    speed_limit: float
+
+
+class Platform(SimulationObject):
+    """A platform where trains can arrive, load and unload passengers and depart"""
+
+    track: Track
+    station_name: str
+    platform_number: int
+    blocked: bool
+
+
+class Train(SimulationObject):
+    """A train driving around in the simulation."""
+
+    # pylint: disable=too-many-instance-attributes
+
+    _position: Tuple[float, float] = None
+    _route: str = None
+    _track: Track = None
+    _vehicle_type: str = None
```

Comment:
Why is vehicle_type needed? It is not used anywhere and seems like a duplicate of train_type.

Reply:
good catch; removed!